src/problems/RadMarshakHot/test_radiation_marshak_hot.cpp

```diff
@@ -0,0 +1,265 @@
+	const double Egas = initial_Egas;
+	consVar(i, j, k, RadSystem<StreamingProblem>::gasEnergy_index) = Egas;
+	consVar(i, j, k, RadSystem<StreamingProblem>::gasDensity_index) = rho;
+	consVar(i, j, k, RadSystem<StreamingProblem>::gasInternalEnergy_index) = Egas;
+	consVar(i, j, k, RadSystem<StreamingProblem>::x1GasMomentum_index) = 0.;
+	consVar(i, j, k, RadSystem<StreamingProblem>::x2GasMomentum_index) = 0.;
+	consVar(i, j, k, RadSystem<StreamingProblem>::x3GasMomentum_index) = 0.;
+}
+
+auto problem_main() -> int
+{
+	// Problem parameters
+	// const int nx = 1000;
+	// const double Lx = 1.0;
+	const double CFL_number = 0.8;
+	const double dt_max = 1e-2;
+	const double tmax = 1.0;
+	const int max_timesteps = 5000;
+
+	// Boundary conditions
+	constexpr int nvars = RadSystem<StreamingProblem>::nvar_;
+	amrex::Vector<amrex::BCRec> BCs_cc(nvars);
+	for (int n = 0; n < nvars; ++n) {
+		BCs_cc[n].setLo(0, amrex::BCType::ext_dir);  // Dirichlet x1
+		BCs_cc[n].setHi(0, amrex::BCType::foextrap); // extrapolate x1
+		for (int i = 1; i < AMREX_SPACEDIM; ++i) {
+			BCs_cc[n].setLo(i, amrex::BCType::int_dir); // periodic
+			BCs_cc[n].setHi(i, amrex::BCType::int_dir);
+		}
+	}
+
+	// Problem initialization
+	QuokkaSimulation<StreamingProblem> sim(BCs_cc);
+
+	sim.radiationReconstructionOrder_ = 3; // PPM
+	sim.stopTime_ = tmax;
+	sim.radiationCflNumber_ = CFL_number;
+	sim.maxDt_ = dt_max;
+	sim.maxTimesteps_ = max_timesteps;
+	sim.plotfileInterval_ = -1;
+
+	// initialize
+	sim.setInitialConditions();
+
+	// evolve
+	sim.evolve();
+
+	// read output variables
+	auto [position, values] = fextract(sim.state_new_cc_[0], sim.Geom(0), 0, 0.0);
+	const int nx = static_cast<int>(position.size());
+
+	// compute error norm
+	std::vector<double> erad(nx);
+	std::vector<double> erad_exact(nx);
+	std::vector<double> T(nx);
+	std::vector<double> xs(nx);
+	for (int i = 0; i < nx; ++i) {
+		amrex::Real const x = position[i];
+		xs.at(i) = x;
+		double erad_sim = 0.0;
+		for (int g = 0; g < Physics_Traits<StreamingProblem>::nGroups; ++g) {
+			erad_sim += values.at(RadSystem<StreamingProblem>::radEnergy_index + Physics_NumVars::numRadVars * g)[i];
+		}
+		erad.at(i) = erad_sim;
+		const double e_gas = values.at(RadSystem<StreamingProblem>::gasInternalEnergy_index)[i];
+		T.at(i) = quokka::EOS<StreamingProblem>::ComputeTgasFromEint(rho, e_gas);
+
+		// compute exact solution
+		const double tau = kappa0 * rho * x;
+		erad_exact.at(i) = (x <= chat * tmax) ? EradL * std::exp(-tau) : 0.0;
+	}
+
+	// double err_norm = 0.;
+	// double sol_norm = 0.;
+	// for (int i = 0; i < nx; ++i) {
+	// 	err_norm += std::abs(erad[i] - erad_exact[i]);
+	// 	sol_norm += std::abs(erad_exact[i]);
+	// }
+
+	// const double rel_err_norm = err_norm / sol_norm;
+	// const double rel_err_tol = 0.02;
+	// int status = 1;
+	// if (rel_err_norm < rel_err_tol) {
+	// 	status = 0;
+	// }
+	// amrex::Print() << "Relative L1 norm = " << rel_err_norm << std::endl;
```

Comment:
Why did you comment this out? I think it's necessary to check the solution here.

Reply:
Updated and now there is an exact solution to compare with.